src/init.cpp

```diff
@@ -1341,8 +1341,9 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         if (!fLoaded) {
             // first suggest a reindex
             if (!fReset) {
-                bool fRet = uiInterface.ThreadSafeMessageBox(
+                bool fRet = uiInterface.ThreadSafeQuestion(
                     strLoadError + ".\n\n" + _("Do you want to rebuild the block database now?"),
+                    strLoadError + ".\nPlease restart with -reindex or -reindex-chainstate to recover.",
```

Comment:
We don't have ``-reindex-chainstate``. That was added in upstream 0.14.0: https://github.com/bitcoin/bitcoin/commit/d3d75479115bc3480f163df774ee9dd2f8bd9f54